sockeye/checkpoint_decoder.py

```diff
@@ -177,7 +177,7 @@ def decode_and_evaluate(self, output_name: Optional[str] = None) -> Dict[str, fl
                     if output_file is not None:
                         print(output_string, file=output_file)
         avg_time = trans_wall_time / len(self.targets_sentences[0])
-        translations = list(zip(*translations))
+        translations = list(zip(*translations))  # type: ignore
```

Comment:
nit: the copyright date of the file could also be updated.

Reply:
sure thing